circleci/Dockerfile

```diff
@@ -50,12 +50,23 @@ RUN set -ex \
     rm -rf /var/lib/apt/lists/*
 
 # Golang
+ARG GO_TOOLCHAIN
+ARG MSGO_VERSION
+ARG MSGO_SHA256_LINUX_AMD64
 ARG GO_VERSION
 ARG GO_SHA256_LINUX_AMD64
+ENV MSGO_VERSION $MSGO_VERSION
```

Comment:
Is this variable needed in the container? (Same question for other occurrences of the
```
ARG FOO
ENV FOO $FOO
```
pattern later)